### PR TITLE
support android single touch mode

### PR DESCRIPTION
**Pull request by pktangyue**
For cocos2dx 2.x version, Android is multi-touch supported by default.

But in some case, multi-touch would cause some unexpected problem, so I just want it only handle one touch at a time.

You can use `Cocos2dxGLSurfaceView.getInstance().setMultiTouchEnabled(false);` to disable multi touch mode.

### CODE REVIEW

**Comment by dumganhar** [comment]
Is `android` providing an API to disable multi-touch?

**Comment by pktangyue** [comment]
I think there is no simple way to disable multi-touch, just like [__glView setMultipleTouchEnabled:YES]; in iOs.

I find [Disabling multitouch in android in stackoverflow](http://stackoverflow.com/questions/12111862/disabling-multitouch-in-android), someone said 'you should turn off [splitMotionEvents](http://developer.android.com/reference/android/R.attr.html#splitMotionEvents) or [windowEnableSplitTouch](http://developer.android.com/reference/android/R.attr.html#windowEnableSplitTouch) attribute inside ViewGroup where buttons are located'.

But I don't think it will work in cocos2d-x, because there is only a GLSurfaceView for android, while these two options only prevent other view to receive touch event to make just only one touch take effect. ( I didn't run and check it. if I'm wrong, point it out)

In my changed code, ACTION_DOWN and ACTION_UP are the states for the first touch point, while ACTION_POINTER_DOWN and ACTION_POINTER_UP are the states for no-first touch points. So I break these two case if set multi-touch false.

---

After more test, I find a bug. And I will send new commit after I fixing it.

**Comment by pktangyue** [comment]
I update a new commit, plz review it.

File: cocos2dx/platform/android/java/src/org/cocos2dx/lib/Cocos2dxGLSurfaceView.java [hunk 1]
```diff
@@ -56,6 +56,7 @@ of this software and associated documentation files (the "Software"), to deal
 	private Cocos2dxRenderer mCocos2dxRenderer;
 	private Cocos2dxEditText mCocos2dxEditText;
 
+	private boolean is_multi_touch_enabled = true;
```
Comment (dumganhar) [inline cocos2dx/platform/android/java/src/org/cocos2dx/lib/Cocos2dxGLSurfaceView.java]: Please use the variable naming style in java. `mXxxXxx`. Thanks.

**Pull request closed without merging**